CoreOpenGLDisplay/src/au/gov/asd/tac/constellation/visual/opengl/utilities/glyphs/GlyphRectangleBuffer.java

```diff
@@ -256,11 +256,12 @@ int addRectImage(final BufferedImage img, final int extra) {
         //
         final int hashCode = Arrays.hashCode(img.getRGB(0, 0, w, h, null, 0, w));
 
-        Integer rectIndex = memory.putIfAbsent(hashCode, memory.size());
+        
+        Integer rectIndex = memory.get(hashCode);
```

Comment:
change to primitive `int`

Reply:
Will it not error if memory.get() returns a null?

Reply:
Right. If the return value can be null then never mind